app/views/submissions/_version_links.html.erb

```diff
@@ -3,15 +3,13 @@
     <%= link_to(url_for([:view, @course, @assessment, @prevVersion[:submission], header_position: @prevVersion[:header_position]]), class: "btn small", title: "Previous version containing this file (shortcut: [)", id: "prev_version_link") do %>
       <i class="material-icons">south</i>
     <% end %>
-    <link rel="prerender" href=<%= url_for([:view, @course, @assessment, @prevVersion[:submission], header_position: @prevVersion[:header_position]]) %> />
```

Comment:
Does this only remove the prerendering or does this also remove the link to the next view?

Reply:
It only removes the prerendering. The actual link is above, inside the `link_to`

Reply:
Have also verified that the links still work